File: Multiplos_de_un_numero/MultiplosDeUnNumero.java
package multiplos_de_un_numero;

import java.util.Arrays;
import java.util.Scanner;

/**
Compilador: Java SE 18.0.2
author: Johnny Hernandez
fecha: 24 de febrero de 2023
*/

public class MultiplosDeUnNumero {

	private int[] function(int numero, int elementos){
		int array[] = new int[elementos];

		for (int x = 0; x < elementos ; x++ ) {
			array[x] = numero * (x + 1);
		}
		return array;
	}

	public static void main(String[] args) {
		MultiplosDeUnNumero myArray = new MultiplosDeUnNumero();

		int numero;
		int elementos;

		Scanner keyboard = new Scanner(System.in);

		System.out.println("Este programa genera un arreglo de multiplos de un numero");

		System.out.print("Ingresar numero: ");
		numero = keyboard.nextInt();
		System.out.print("Ingresar cantidad de elementos: ");
		elementos = keyboard.nextInt();

		System.out.print("Arreglo de multiplos de " + numero + " es " +
                Arrays.toString(myArray.function(numero, elementos)));
	}
}

File: Multiplos_de_un_numero/multiplos_de_un_numero.py
#!/usr/bin/env python3

'''
Compilador: Python 3.10.4
author: Johnny Hernandez
fecha: 24 de febrero de 2023
'''

def function(numero: int, elementos: int):
	array = []
	for x in range(1, elementos + 1):
		array.append(numero * x)
	return array

def main():
	print("Este programa genera un arreglo de multiplos de un número\n")
	numero = int(input("Ingresar número: "))
	elementos = int(input("Ingresar cantidad de elementos: "))

	print("Arreglo de multiplos de ", numero, " es ", function(numero, elementos))

if __name__ == '__main__':
	main()
